pcf/particle/aws/glacier/glacier_vault.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from botocore.exceptions import ClientError
+from pcf.core.aws_resource import AWSResource
+from pcf.core import State
+
+logger = logging.getLogger(__name__)
+
+class GlacierVault(AWSResource):
+    """
+    This is the implementation of Amazon's Glacier service. Additional functions that are callable on this particle
+    are upload_archive, initiate_job, list_jobs, delete_archive, delete_job, add_tags_to_vault, list_tags_for_vault
+    """
+    flavor = "glacier"
+
+    state_lookup = {
+        "missing": State.terminated,
+        "active": State.running,
+        "inactive": State.terminated
+    }
+
+    equivalent_states = {
+        State.running: 1,
+        State.stopped: 0,
+        State.terminated: 0
+    }
+
+    def __init__(self, particle_definition, session=None):
+        super(GlacierVault, self).__init__(
+            particle_definition=particle_definition,
+            resource_name="glacier",
+            session=session
+        )
+        self.vault_name = self.desired_state_definition.get("vaultName")
+        self.account_id = self.desired_state_definition.get("accountId", "-")
+
+    def _terminate(self):
+        """
+        Deletes the Glacier vault if vault is empty
+
+        Returns:
+             response of boto3 delete_vault
+        """
+        response = self.client.delete_vault(vaultName=self.vault_name)
+        message = "hey"
```

Comment:
dont need a "hey" message

Reply:
Sorry about that. I forgot to remove that.